pipelines/jobs/configurations/jdk16_pipeline_config.groovy

```diff
@@ -105,6 +105,18 @@ class Config16 {
                 ],
                 configureArgs       : '--enable-dtrace'
         ],
+
+        arm32Linux    : [
+                os                  : 'linux',
+                arch                : 'arm',
+                // TODO Temporarily remove the ARM tests because we don't have fast enough hardware
```

Comment:
Can that commented out code go now?

Reply:
Since we're testing arm on multiple systems now, yeah probably